### DISCUSSION TITLE
Restrict the creation of subpages

### DISCUSSION THREAD

**Discussion by PubliAlex** [OFFLINE-GmbH/oc-boxes-support#182] [Ideas and feature requests]
Hello,

I work on a website using boxes where the customer can create pages, but I would like to restrict the creation of subpage, would it be possible to add an option for that ?

Thank you

**Comment by tobias-kuendig** [comment]
Is there still a need for creating subpages on your site or do you want to limit the possible levels of subpages globally?

Adding a permission to disallow some users to create subpages sounds very specific to your project, so it would not fit well in the plugin. Limiting the allowed levels of pages, however, could be a more commonly useful feature. You could set your allowed levels to `1`.

Would that fit for your use-case?

**Comment by PubliAlex** [comment]
Hello,

In my case, it's global, the theme just don't support subpages, so a global settings where you can define the maximum level of subpage you can create is ok and can be useful for many use cases cause not all design system support indefinite level in a hierarchical menu

**Reply by tobias-kuendig** [reply]
A new option to limit the max. of allowed levels has been added in v3.6.48.

**Comment by PubliAlex** [comment]
@tobias-kuendig thanks, you are the boss 👍
